hack/multi-node/Vagrantfile

```diff
@@ -127,6 +127,7 @@ Vagrant.configure("2") do |config|
 
       controller.vm.provision :file, source: CONTROLLER_USER_DATA_PATH, destination: "/tmp/vagrantfile-user-data"
       controller.vm.provision :shell, inline: "mv /tmp/vagrantfile-user-data /var/lib/coreos-vagrant/", privileged: true
+      controller.vm.provision :shell, inline: "mkdir /etc/kubernetes/manifests", privileged: true
```

Comment:
Can we move this into an ExecStartPre in the kubelet unit. It works here, but at least we're describing what is necessary in the systemd unit in a more portable way.